README.md

```diff
@@ -1,6 +1,7 @@
 This README is under construction as we work to build a new community driven high performance key-value store.
 
 This project was forked from the open source Redis project right before the transition to their new source available licenses.
+Redis is an in-memory database that persists on disk. The data model is key-value, but many different kind of values are supported: Strings, Lists, Sets, Sorted Sets, Hashes, Streams, HyperLogLogs, Bitmaps.
```

Comment:
```suggestion
```

Reply:
To your point, maybe do this then?

Reply:
Correct me if I'm wrong, but after I found out what Redis did, that told me what valkey does. 
So why would anybody suggest deleting that portion of the text that literally told me what valkey does.  I think adding a little bit of context is a good thing.  
On line 4, just delete the "Redis" word and replace it with the "valkey" word.  Conflict and difficult "train of thought" over.

Reply:
There is a section a little bit further down that says "What is Valkey though". You can add more information there if you like.